addons/skin.estuary/xml/Includes_PVR.xml

```diff
@@ -246,23 +246,31 @@
 				<width>830</width>
 				<height>262</height>
 				<font>font36_title</font>
-				<label>$INFO[ListItem.Title]</label>
+				<label>$INFO[ListItem.Title] $INFO[ListItem.Year,([COLOR grey],[/COLOR])]</label>
 				<visible>!PVR.HasEpg</visible>
 			</control>
 			<control type="label">
 				<top>390</top>
 				<width>830</width>
 				<height>262</height>
 				<font>font36_title</font>
-				<label>$INFO[ListItem.EpgEventTitle]</label>
+				<label>$INFO[ListItem.EpgEventTitle] $INFO[ListItem.Year,([COLOR grey],[/COLOR])]</label>
 				<visible>PVR.HasEpg</visible>
 			</control>
 			<control type="label">
 				<top>435</top>
-				<width>830</width>
+				<width>40%</width>
 				<height>70</height>
 				<label>[I]$VAR[SeasonEpisodeLabel]$INFO[ListItem.EpisodeName][/I]</label>
 			</control>
+			<control type="label">
+				<top>435</top>
+				<right>30</right>
+				<width>60%</width>
+				<height>70</height>
+				<align>right</align>
+				<label>$INFO[ListItem.Genre,[COLOR grey]$LOCALIZE[515]:[/COLOR] ]</label>
+			</control>
```

Comment:
I would place the genre where date, duration, etc. is placed, looking similar to what you did for the info dialog. this seems to be more consistent.

Reply:
Ah, I was imitating how it was done on the guide screen (as they are all the same depth), episode on left and genre on right.

Reply:
Episode is under title on the info screen so different to all other screens.

Reply:
Fair enough, so it might make sense to change the info dialog to match your changes.

Reply:
I thought about that but the fact that it’s full screen vs a side pane I’m not sure which would be better.

Reply:
Or maybe it’s fine as is.

Reply:
I have no strong opinion about that.